src/client/components/Buttons/Buttons.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+
+const Buttons = (props) => {
```

Comment:
What is this doing? If its part of another feature, lets break it out into its own ticket